packages/terra-card/src/Card.scss

```diff
@@ -0,0 +1,27 @@
+
+/* stylelint-disable selector-class-pattern  */
+.card {
+  background-color: #fff;
+  border-radius: 3px;
+  box-shadow: 0 1px 1px 0 #1c1f21;
+  margin: 10px;
```

Comment:
I remember from Friday's conversation that there wouldn't be any margin on the card but would be on the responsibility of card-collection or similar grid based parent. Is that a correction assumption of always the containing component handling layout/placement?

It still could be a themeable option, I suppose in addition, but would need to investigate/validate further.